docs/docs/api/Tasks/createTasks.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Create Task
+---
+
+export const Endpoint = ({children, color}) => ( <span style={{
+borderRadius: '2px',
+color: '#E83E8C',
+}}>{children}</span> );
+
+<Endpoint>POST tasks/</Endpoint>: Create a task with provided task details <br></br>
```

Comment:
Minor comment: the `/` should be before `tasks` not after (keeps consistency with the other routes documented as well)

Reply:
Fixed: Changes are reflected in recent commits